docusaurus/docusaurus.config.js

```diff
@@ -286,6 +286,8 @@ const config = {
           alt: 'The OpenZiti logo, an open source zero trust network overlay',
           src: 'img/ziti-logo-dark.svg',
           srcDark: 'img/ziti-logo-light.svg',
+          href: 'https://openziti.io/',
```

Comment:
Allows the button to work wherever the site is hosted, e.g., locally or Vercel, too.

```suggestion
          href: '/',
```

Reply:
please put a branch up for this and verify it works. i want to commit what we have as is to make sure this process works

Reply:
I tested it locally, but I can send a twig PR too.

Reply:
twig PR: https://github.com/openziti/ziti-doc/pull/629